db/schema.rb

```diff
@@ -743,6 +751,7 @@
     t.boolean "allow_welsh_translation", default: false, null: false
     t.boolean "allow_multiple_proceedings", default: false, null: false
     t.boolean "enable_ccms_submission", default: true, null: false
+    t.boolean "alert_via_sentry", default: false, null: false
```

Comment:
This looks like it might have crept in from another PR.

Reply:
That was part of the sentry alternative PR I did before I went away.  Shouldnt' be part of this PR